static/dbview.config.php

```diff
@@ -0,0 +1,234 @@
+			"location" => ["contact", "location"],
+			"about" => ["contact", "about"],
+			"keywords" => ["contact", "keywords"],
+			"gender" => ["contact", "gender"],
+			"xmpp" => ["contact", "xmpp"],
+			"attag" => ["contact", "attag"],
+			"avatar" => ["contact", "avatar"],
+			"photo" => ["contact", "photo"],
+			"thumb" => ["contact", "thumb"],
+			"micro" => ["contact", "micro"],
+			"site-pubkey" => ["contact", "site-pubkey"],
+			"issued-id" => ["contact", "issued-id"],
+			"dfrn-id" => ["contact", "dfrn-id"],
+			"url" => ["contact", "url"],
+			"nurl" => ["contact", "nurl"],
+			"addr" => ["contact", "addr"],
+			"alias" => ["contact", "alias"],
+			"pubkey" => ["contact", "pubkey"],
+			"prvkey" => ["contact", "prvkey"],
+			"batch" => ["contact", "batch"],
+			"request" => ["contact", "request"],
+			"notify" => ["contact", "notify"],
+			"poll" => ["contact", "poll"],
+			"confirm" => ["contact", "confirm"],
+			"poco" => ["contact", "poco"],
+			"aes_allow" => ["contact", "aes_allow"],
+			"ret-aes" => ["contact", "ret-aes"],
+			"usehub" => ["contact", "usehub"],
+			"subhub" => ["contact", "subhub"],
+			"hub-verify" => ["contact", "hub-verify"],
+			"last-update" => ["contact", "last-update"],
+			"success_update" => ["contact", "success_update"],
+			"failure_update" => ["contact", "failure_update"],
+			"name-date" => ["contact", "name-date"],
+			"uri-date" => ["contact", "uri-date"],
+			"avatar-date" => ["contact", "avatar-date"],
+			"picdate" => ["contact", "avatar-date"], /// @todo Replaces all uses of "picdate" with "avatar-date"
+			"term-date" => ["contact", "term-date"],
+			"last-item" => ["contact", "last-item"],
+			"priority" => ["contact", "priority"],
+			"blocked" => ["contact", "blocked"], /// @todo Check if "blocked" from contact or from the users table
+			"block_reason" => ["contact", "block_reason"],
+			"readonly" => ["contact", "readonly"],
+			"writable" => ["contact", "writable"],
+			"forum" => ["contact", "forum"],
+			"prv" => ["contact", "prv"],
+			"contact-type" => ["contact", "contact-type"],
+			"hidden" => ["contact", "hidden"],
+			"archive" => ["contact", "archive"],
+			"pending" => ["contact", "pending"],
+			"deleted" => ["contact", "deleted"],
+			"rating" => ["contact", "rating"],
+			"unsearchable" => ["contact", "unsearchable"],
+			"sensitive" => ["contact", "sensitive"],
+			"baseurl" => ["contact", "baseurl"],
+			"reason" => ["contact", "reason"],
+			"closeness" => ["contact", "closeness"],
+			"info" => ["contact", "info"],
+			"profile-id" => ["contact", "profile-id"],
+			"bdyear" => ["contact", "bdyear"],
+			"bd" => ["contact", "bd"],
+			"notify_new_posts" => ["notify_new_posts"],
+			"fetch_further_information" => ["fetch_further_information"],
+			"ffi_keyword_blacklist" => ["ffi_keyword_blacklist"],
+			"parent-uid" => ["user", "parent-uid"],
+			"guid" => ["user", "guid"],
+			"nickname" => ["user", "nickname"], /// @todo Replaces all uses of "nickname" with "nick"
```

Comment:
Why this TODO?

Reply:
We are using the "username" element a lot. But we store the name also in the array element "name", so this is a duplicate.

Reply:
"name" or "nick" ?